compaction/compaction.cc

```diff
@@ -1178,7 +1204,6 @@ class regular_compaction : public compaction {
 };
 
 class cleanup_compaction final : public regular_compaction {
-    mutable dht::incremental_owned_ranges_checker _owned_ranges_checker;
```

Comment:
I guess the mutable was moved from here. So you don't have to fix it now if that's the case, but it's still suspicious.

Reply:
exactly

Reply:
gone in v2